packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -222,6 +245,16 @@ export function ArticleMetadataPanel({
                     setAuthorsFilter('')
                   }}
                   block
+                  renderExtraFooter={() =>
+                    authorsFilter &&
+                    !data?.authors.nodes.length && (
+                      <div style={{margin: '20px'}}>
+                        <Button onClick={() => handleCreateAuthor()} appearance="primary">
+                          {t('articles.panels.createAuthorProfile', {name: authorsFilter})}
```

Comment:
It's a nice touch to add the user name to the comment, but as a personal preference I find it distracting that the button is dynamically changing size. Maybe the comment can be in a label and the button should have a static text "Create"?

Reply:
I think it's good if the name is in the button. Otherwise the users might think it will open the `AuthorEditPanel` and that's not what happens.